frontends/mit-open/src/pages/HomePage/TestimonialsSection.tsx

```diff
@@ -196,6 +203,11 @@ const SlickCarousel = () => {
     ],
   }
 
+  const generateMarketingImageSrc = () => {
+    const idx = Math.floor(Math.random() * 6) + 1
```

Comment:
Is there something we can do here to ensure that two of the same images don't land directly next to each other? This happened to me and of course I didn't get a screenshot of it, but it didn't look great to have the exact same images on two cards... at least side by side.

Reply:
I added some code to do this - you will still get wraparound but it shouldn't drop one next to each other. The algorithm is pretty simple but can also set it up so that you don't get a repeat until all of them have been seen once.

Reply:
Thanks, that makes sense. We can always add more marketing images too... as long as we have more images than attestations then we'll never see a repeat.